data/root/root.file_list

```diff
@@ -448,12 +448,7 @@ if arch ne 's390' && arch ne 's390x'
   ?xf86-input-vmmouse:
   xf86-input-void:
   xf86-input-wacom:
-  xf86-video-ati: nodeps
-  xf86-video-cirrus:
```

Comment:
See the discussion in fate #323161. Besides, the point is moot, the packages are already gone. This is just to make installation-images build again.